ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/RewardsActionsComposable.kt

```diff
@@ -74,38 +76,33 @@ fun ActionCard(
 ) {
   Card(
     modifier = Modifier
-      .padding(
-        start = 8.dp,
-        top = 16.dp
-      )
-      .width(166.dp)
-      .height(200.dp)
+      .size(width = 160.dp, height = 208.dp)
       .clickable { onClick() },
     shape = RoundedCornerShape(8.dp),
     colors = CardDefaults.cardColors(WalletColors.styleguide_blue_secondary),
   ) {
     Column(
-      modifier = Modifier
-        .padding(16.dp)
-        .fillMaxSize(),
-      verticalArrangement = Arrangement.SpaceEvenly,
+      modifier = Modifier.padding(16.dp),
+      verticalArrangement = Arrangement.spacedBy(16.dp),
       horizontalAlignment = Alignment.CenterHorizontally
     ) {
       Image(
         painter = painterResource(image),
-        "Earn Money",
+        contentDescription = null,
         modifier = Modifier
           .height(54.dp)
           .width(54.dp)
       )
       Text(
         text = stringResource(id = title),
-        style = MaterialTheme.typography.titleMedium,
-        color = WalletColors.styleguide_white
+        style = MaterialTheme.typography.bodyMedium,
```

Comment:
Why didn't you use `titleSmall` typography? They only differ in letter spacing

Reply:
you're right regarding titlesmall, I alrealdy changed it